src/containers/Permissions.jsx

```diff
@@ -31,6 +31,7 @@ const Permissions = () => {
     as: KONNECTORS_DOCTYPE,
     fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
   })
+  const toNotDisplay = ['Home', 'Store', 'Settings']
```

Comment:
let's add those 3 applications inside queryAppsResult inside the tests, to check they are not present inside the DOM